README.rst

```diff
@@ -21,6 +21,8 @@
 	   :target: https://github.com/psf/black
 .. image:: https://coveralls.io/repos/github/Happy-Algorithms-League/python-gp/badge.svg?branch=master
 	   :target: https://coveralls.io/github/Happy-Algorithms-League/python-gp?branch=master
+.. image:: https://readthedocs.org/projects/ansicolortags/badge/?version=latest
```

Comment:
does this badge break if the docs are not built correctly?

Reply:
No, this badge is "stolen" from readthedocs, so it will not change, it's a static picture. However, we're testing the documentation build in CI, so we'll (almost certainly) never have breaking documentation on master.

Reply:
excellent, thanks!